packages/discord-utilities/src/lib/limits.ts

```diff
@@ -137,7 +140,40 @@ export const MessageLimits = {
 	/**
 	 * Maximum characters allowed in a single message for a nitro user.
 	 */
-	MaximumNitroLength: 4000
+	MaximumNitroLength: 4000,
+
+	/**
+	 * Maximum numbers of reactions allowed for a message.
+	 */
+	MaximumReactions: 20,
+
+	/**
+	 * Maximum upload size for a free user in a guild of tier 1 or below, or in DMs.
+	 * Size is in bytes, and correspond to 8MB.
+	 */
+	MaximumUploadSize: 8_000_000,
+
+	/**
+	 * Maximum upload size for a nitro user, in any guild or in DMs.
+	 * Size is in bytes, and correspond to 10MB.
+	 */
+	MaximumNitroUploadSize: 100_000_000,
+
+	/**
+	 * Maximum upload size for a free user for all different boost levels available in a guild.
+	 * Sizes are in bytes, and correspond to 8MB, 8MB, 50MB, and 100MB.
+	 */
+	MaximumUploadSizeInGuild: [8_000_000, 8_000_000, 50_000_000, 100_000_000]
```

Comment:
Clever

Reply:
Useful if you want to get the one corresponding to your guild, you can use the tier as an index ^^